File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

//validate input

    int height;

    height = -1;

    while (height < 0 || height > 23)
    {

        height = get_int("Positive Number: ");
    }

// write spaces and hashes

    for (int i = 1; i <= height; i++)
    {

        // spaces

        for (int j = 0; j < height - i; j++)
        {

            printf(" ");

        }

        // hashes

        for (int k = 1; k <= i; k++)
        {

            printf("#");

        }

        // space

        printf(" ");

        //hashes

        for (int k = 1; k <= i; k++)
        {

            printf("#");
        }

        //spaces

       /* for (int j = 0; j < (height - 1); j++)
        {

            printf(" ");

        }*/


        //new line

        printf("\n");



    }

}

File: pset6/mario/mario.py
import cs50

height = -1

while height < 0 or height > 23:
    print("please enter a height:")
    height = cs50.get_int()

spaces = height - 1
hashes = 1
b = 0

while b < height:
    print(' ' * spaces, end="")
    print('#' * hashes, end="")
    print("  ", end="")
    print('#' * hashes, end="")
    print()
    b = b + 1
    spaces = spaces - 1
    hashes = hashes + 1